pydicom/filereader.py

```diff
@@ -233,7 +233,7 @@ def data_element_generator(fp, is_implicit_VR, is_little_endian,
         # Reading the value
         # First case (most common): reading a value with a defined length
         if length != 0xFFFFFFFF:
-            if defer_size is not None and length > defer_size:
+            if defer_size is not None and length > defer_size and tag != (0x08, 0x05):
```

Comment:
Can you add a comment noting why the tag check is necessary (maybe along the lines of the tag is used immediately so must return a non `None` value).